spec/03-plugins/23-rate-limiting/04-access_spec.lua

```diff
@@ -761,7 +761,8 @@ if limit_by == "ip" then
 
     assert
       .with_timeout(15)
-      .with_max_tries(10)
+      .with_step(0.5) -- the windows is 1 second, we wait 0.5 seconds between each retry,
+                      -- that can avoid some unlucky case (we are at the end of the window)
```

Comment:
By default the `step` is `.0.05` which has better control than `0.5`.

We can either increase the `with_timeout` or `with_max_tries`.

Reply:
If `tries` is exceeded but not timeout, the assertion will fail. So increasing the timeout doesn't work.

Reply:
Seems there is not a good way to it. We can increase both values. The corner case is the CI takes slightly longer.

Reply:
fixed